tools/cellxgene_census_builder/src/cellxgene_census_builder/host_validation.py

```diff
@@ -70,28 +70,25 @@ def check_host(args: CensusBuildArgs) -> bool:
 
     return (
         check_os()
-        and check_physical_memory(
-            args.config.get("min_physical_memory", args.config.host_validation_min_physical_memory)
```

Comment:
Not sure where these args were coming from. I searched and they don't appear anywhere in the project.

Reply:
looks like a latent bug you have caught...nice